.github/workflows/continuous-integration.yml

```diff
@@ -580,6 +631,209 @@ jobs:
           path: cypress/results
           retention-days: 1
 
+  stress-test-cypress-tests:
```

Comment:
Should this job along with `update-e2e-allow-list` be prevented from running on the `main` branch? Since this will run in CI for PRs, flaky tests won't be able to make it to `main`, and won't be run in the `cypress-tests` job. So I'm thinking we shouldn't need to stress test on `main`?

Reply:
We've decided to stress-test e2e tests against `main` because it can't hurt. And if a test flakes, we'll disabled it. There are no downsides to this as far as we can tell.

Reply:
> And if a test flakes, we'll disabled it. There are no downsides to this as far as we can tell.

Just checking, would that open up a PR to fix the  flaky test now disabled in main?

Reply:
It will update the disallow list to include the flaky test file. The team will get a ticket indicating there is an issue with their test spec. It will be up to the team to generate a PR to fix the disallowed test file.

Reply:
No, it creates a GitHub issue labeled with the team label as noted in the
product directory.